src/app/Main/Header/Export/Dialog/Header-ExportDialog.component.tsx

```diff
@@ -32,36 +32,44 @@ export function HeaderExportDialogComponent(props: {
 
     const handleExport = (): void => props.onClickExport();
 
-    const fileIsSelect = props.settings.raw || props.settings.stat;
+    const handleClearData = (): void => props.clearData();
+
+    const fileIsSelect = props.raw || props.stat;
 
     const exportDialogTitle = t("Content.Title");
     const exportContentText = t("Content.Text");
     const exportCheckboxes = (
         <div>
             <ExportDialogCheckboxesComponent
                 onCheckboxChange={handleChangeCheckbox}
-                isRaw={props.settings.raw}
-                isStat={props.settings.stat}
+                isRaw={props.raw}
+                isStat={props.stat}
             />
             {!fileIsSelect && <p css={warningStyle}>{t("Warning")}</p>}
         </div>
     );
 
     const exportCancelButton = t("Button.Cancel");
 
-    const exportSubmitButton = (
-        <ButtonsCsvLinkComponent
-            setting={props.settings}
-            filter={props.filter.selectedFilter}
-            mainFilterAttributes={props.filter.mainFilter}
-            buttonLabel={props.buttonLabel}
-            ZNFilename={props.exportLabels.ZNFilename}
-            mainFilterLabels={props.exportLabels.mainFilterLabels}
-            allFilterLabel={props.exportLabels.allFilterLabel}
-        />
-    );
+    const exportButtonLabel = props.buttonLabel;
+    let exportSubmitButton: JSX.Element = exportButtonLabel;
+
+    if (props.dataString !== undefined) {
+        exportSubmitButton = (
+            <div>
+                {exportButtonLabel}
+                <CSVDownload
+                    data={props.dataString}
+                    filename={props.ZNFilename}
+                    target="_blank"
+                />
+            </div>
+        );
+        handleClearData()
```

Comment:
I really don't get this, if the dialog is called with a download link, it gets closed and the link is erased? How does this work?